core/src/main/python/wlsdeploy/tool/deploy/applications_deployer.py

```diff
@@ -9,10 +9,25 @@
 import os
 from java.io import File
 from java.io import IOException
+from java.io import FileOutputStream
+from java.lang import StringBuilder
+from java.io import BufferedReader
+from java.io import BufferedWriter
+from java.io import FileInputStream
+from java.io import FileOutputStream
+from java.io import InputStreamReader
+from java.io import OutputStreamWriter
```

Comment:
FileOutputStream is in the list twice.
These don't seem to be used: StringBuilder, BufferedReader, BufferedWriter, FileInputStream, InputStreamReader, OutputStreamWriter